runtime/near-vm-logic/src/logic.rs

```diff
@@ -609,6 +609,36 @@ impl<'a> VMLogic<'a> {
         Ok(self.context.epoch_height)
     }
 
+    /// Returns the stake of an account, if the account is currently a validator. Otherwise returns 0.
```

Comment:
It should explain what is the size of `stake` that it writes into the memory, see example: https://nomicon.io/RuntimeSpec/Components/BindingsSpec/EconomicsAPI.html

> -- writes the value into the u128 variable pointed by balance_ptr.

Same below.